--- kubejs/server_scripts/advjs_cl5.js.bak	2024-04-27 12:32:55.418389500 +0200
+++ kubejs/server_scripts/advjs_cl5.js	2024-04-27 12:59:03.685188200 +0200
@@ -22,8 +22,8 @@
     .create("advjs:cl5")
     .display((displayBuilder) => {
       displayBuilder.setIcon("create_dd:integrated_mechanism");
-      displayBuilder.setTitle("Create Live 5");
-      displayBuilder.setDescription("Willkommen zu Create Live 5!");
+      displayBuilder.setTitle({translate: 'cl5.advjs.welcome.title'});
+      displayBuilder.setDescription({translate: 'cl5.advjs.welcome.description'});
       displayBuilder.setBackground(
         "textures/gui/advancements/backgrounds/adventure.png"
       );
@@ -57,10 +57,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("cobblestone");
-        displayBuilder.setTitle(Text.translate("Stone Age"));
-        displayBuilder.setDescription(
-          Text.translate("Der Stein kommt ins Rollen")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.cobblestone.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.cobblestone.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -74,8 +72,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:charcoal");
-        displayBuilder.setTitle(Text.translate("Kohle, Kohle, Kohle"));
-        displayBuilder.setDescription(Text.translate("Brennt besser als Holz"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.charcoal.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.charcoal.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -94,10 +92,8 @@
       advBuilder
         .display((displayBuilder) => {
           displayBuilder.setIcon("minecraft:sunflower");
-          displayBuilder.setTitle(Text.translate("Shred the Sun"));
-          displayBuilder.setDescription(
-            Text.translate("Vielseitiger als Du denkst")
-          );
+          displayBuilder.setTitle({translate: 'cl5.advjs.item.crushed_sunflower_seeds.title'});
+          displayBuilder.setDescription({translate: 'cl5.advjs.item.crushed_sunflower_seeds.description'});
           displayBuilder.setHidden(false);
         })
         .criteria((criteriaBuilder) => {
@@ -115,10 +111,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("travelersbackpack:standard");
-        displayBuilder.setTitle(Text.translate("Bigger Inventory"));
-        displayBuilder.setDescription(
-          Text.translate("Mehr Platz zum mitnehmen bitte!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.custom_travelers_backpack.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.custom_travelers_backpack.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -179,8 +173,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:iron_ingot");
-        displayBuilder.setTitle(Text.translate("Iron Robbery"));
-        displayBuilder.setDescription(Text.translate("Gib das her!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.iron_ingot.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.iron_ingot.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -202,8 +196,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:cog_crank");
-        displayBuilder.setTitle(Text.translate("Human Energy"));
-        displayBuilder.setDescription(Text.translate("Hände an die Cogs!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.cog_crank.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.cog_crank.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -220,8 +214,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:millstone");
-        displayBuilder.setTitle(Text.translate("Stone Miller"));
-        displayBuilder.setDescription(Text.translate("Gib mal den Grinder"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.millstone.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.millstone.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -238,8 +232,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:sand");
-        displayBuilder.setTitle(Text.translate("Sandy"));
-        displayBuilder.setDescription(Text.translate("Kein Sand im Getriebe"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.sand.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.sand.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -253,8 +247,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:redstone");
-        displayBuilder.setTitle(Text.translate("Stone, but Red"));
-        displayBuilder.setDescription(Text.translate("Nein Jo, nicht deins!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.redstone.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.redstone.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -271,8 +265,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:asurine");
-        displayBuilder.setTitle(Text.translate("Asurine"));
-        displayBuilder.setDescription(Text.translate("Sonnenstein"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.asurine.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.asurine.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -286,8 +280,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:calcite");
-        displayBuilder.setTitle(Text.translate("Calcium"));
-        displayBuilder.setDescription(Text.translate("Gut für die Zähne"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.calcite.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.calcite.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -304,10 +298,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:quartz");
-        displayBuilder.setTitle(Text.translate("Wait! What?"));
-        displayBuilder.setDescription(
-          Text.translate("War wohl reiner Quarzsand")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.quartz.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.quartz.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -320,10 +312,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:andesite_alloy");
-        displayBuilder.setTitle(Text.translate("Finally!"));
-        displayBuilder.setDescription(
-          Text.translate("Die Ära der Technik beginnt!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.andesite_alloy.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.andesite_alloy.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -341,8 +331,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:andesite_casing");
-        displayBuilder.setTitle(Text.translate("Tier 1 Machinery"));
-        displayBuilder.setDescription(Text.translate("Zeit für Maschinen!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.andesite_casing.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.andesite_casing.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("goal");
       })
@@ -360,10 +350,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:water_wheel");
-        displayBuilder.setTitle(Text.translate("Hydropower!"));
-        displayBuilder.setDescription(
-          Text.translate("Nutze die Kraft des Wassers")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.water_wheel.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.water_wheel.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -380,8 +368,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:white_sail");
-        displayBuilder.setTitle(Text.translate("Power of the wind!"));
-        displayBuilder.setDescription(Text.translate("Segel in den Wind!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.white_sail.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.white_sail.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -399,8 +387,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:steam_engine");
-        displayBuilder.setTitle(Text.translate("High Energy"));
-        displayBuilder.setDescription(Text.translate("James Watt wäre Stolz!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.steam_engine.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.steam_engine.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -418,10 +406,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_new_age:generator_coil");
-        displayBuilder.setTitle(Text.translate("Electricity!"));
-        displayBuilder.setDescription(
-          Text.translate("Strom aus kinetischer Energie!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.generator_coil.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.generator_coil.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -444,10 +430,8 @@
       advBuilder
         .display((displayBuilder) => {
           displayBuilder.setIcon("create_new_age:electrical_connector");
-          displayBuilder.setTitle(Text.translate("Electrical Connection"));
-          displayBuilder.setDescription(
-            Text.translate("Der Beginn eines Stromnetzes!")
-          );
+          displayBuilder.setTitle({translate: 'cl5.advjs.item.electrical_connector.title'});
+          displayBuilder.setDescription({translate: 'cl5.advjs.item.electrical_connector.description'});
           displayBuilder.setHidden(false);
         })
         .criteria((criteriaBuilder) => {
@@ -467,10 +451,8 @@
       advBuilder
         .display((displayBuilder) => {
           displayBuilder.setIcon("create_new_age:basic_motor");
-          displayBuilder.setTitle(Text.translate("Electric Motor!"));
-          displayBuilder.setDescription(
-            Text.translate("Kinetische Energie aus Strom!")
-          );
+          displayBuilder.setTitle({translate: 'cl5.advjs.item.basic_motor.title'});
+          displayBuilder.setDescription({translate: 'cl5.advjs.item.basic_motor.description'});
           displayBuilder.setHidden(false);
         })
         .criteria((criteriaBuilder) => {
@@ -488,8 +470,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_new_age:basic_solar_heating_plate");
-        displayBuilder.setTitle(Text.translate("Solar Energy"));
-        displayBuilder.setDescription(Text.translate("Sonne macht Heiß!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.basic_solar_heating_plate.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.basic_solar_heating_plate.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -506,10 +488,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_new_age:heat_pipe");
-        displayBuilder.setTitle(Text.translate("Heat Pipes"));
-        displayBuilder.setDescription(
-          Text.translate("Wir nehmen die Wärme und schieben sie wo anders hin!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.heat_pipe.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.heat_pipe.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -526,8 +506,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_new_age:heat_pump");
-        displayBuilder.setTitle(Text.translate("Pump the Heat"));
-        displayBuilder.setDescription(Text.translate("Auch Hitze kann pumpen"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.heat_pump.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.heat_pump.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -544,10 +524,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_new_age:heater");
-        displayBuilder.setTitle(Text.translate("Better than Blazes"));
-        displayBuilder.setDescription(
-          Text.translate("Blaze Burner sind Steinzeit!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.heater.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.heater.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -564,8 +542,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_connected:shear_pin");
-        displayBuilder.setTitle(Text.translate("Mechanical Fuse"));
-        displayBuilder.setDescription(Text.translate("Shaftschutzschalter"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.shear_pin.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.shear_pin.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -582,10 +560,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:brass_casing");
-        displayBuilder.setTitle(Text.translate("Tier 2 Machinery"));
-        displayBuilder.setDescription(
-          Text.translate("Bessere Maschinen dank Messing!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.brass_casing.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.brass_casing.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("goal");
       })
@@ -603,10 +579,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("toms_storage:ts.storage_terminal");
-        displayBuilder.setTitle(Text.translate("High-End Storage"));
-        displayBuilder.setDescription(
-          Text.translate("Fast zu schön um wahr zu sein")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.storage_terminal.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.storage_terminal.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -625,8 +599,8 @@
         displayBuilder.setIcon(
           "create_mechanical_extruder:mechanical_extruder"
         );
-        displayBuilder.setTitle(Text.translate("Auto-Cobble"));
-        displayBuilder.setDescription(Text.translate("Besser als Drills"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.mechanical_extruder.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.mechanical_extruder.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -643,10 +617,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:deforester_saw");
-        displayBuilder.setTitle(Text.translate("Bye Rainforest"));
-        displayBuilder.setDescription(
-          Text.translate("Entwaldung für Fortgeschrittene")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.deforester_saw.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.deforester_saw.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -663,8 +635,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:item_vault");
-        displayBuilder.setTitle(Text.translate("Item Vault"));
-        displayBuilder.setDescription(Text.translate("Speicher ohne Ende!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.item_vault.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.item_vault.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -678,8 +650,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:chromatic_compound");
-        displayBuilder.setTitle(Text.translate("High-Tier Material"));
-        displayBuilder.setDescription(Text.translate("Ab jetzt wirds teuer"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.chromatic_compound.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.chromatic_compound.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("goal");
       })
@@ -697,10 +669,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_mechanical_spawner:mechanical_spawner");
-        displayBuilder.setTitle(Text.translate("Custom Spawner"));
-        displayBuilder.setDescription(
-          Text.translate("Spawner? Bau ich mir selber!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.mechanical_spawner.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.mechanical_spawner.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -719,8 +689,8 @@
         displayBuilder.setIcon(
           "create_mechanical_spawner:spawn_fluid_random_bucket"
         );
-        displayBuilder.setTitle(Text.translate("Spawner Fluid"));
-        displayBuilder.setDescription(Text.translate("4free ist das nicht!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.spawn_fluid_random_bucket.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.spawn_fluid_random_bucket.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -739,10 +709,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:crushing_wheel");
-        displayBuilder.setTitle(Text.translate("Advanced Crushing"));
-        displayBuilder.setDescription(
-          Text.translate("Grinder für Fortgeschrittene")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.crushing_wheel.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.crushing_wheel.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -761,10 +729,8 @@
         displayBuilder.setIcon(
           "create_tweaked_controllers:tweaked_linked_controller"
         );
-        displayBuilder.setTitle(Text.translate("Gamecontroller"));
-        displayBuilder.setDescription(
-          Text.translate("Hardware trifft auf Software")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.tweaked_linked_controller.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.tweaked_linked_controller.description'});
         displayBuilder.setHidden(true);
       })
       .criteria((criteriaBuilder) => {
@@ -783,8 +749,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:stargaze_singularity");
-        displayBuilder.setTitle(Text.translate("Stargaze Singularity"));
-        displayBuilder.setDescription(Text.translate("Per aspera ad astra"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.stargaze_singularity.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.stargaze_singularity.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("goal");
       })
@@ -802,8 +768,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("toms_storage:ts.crafting_terminal");
-        displayBuilder.setTitle(Text.translate("Nearly ME System"));
-        displayBuilder.setDescription(Text.translate("Lagersystem Endstufe"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.crafting_terminal.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.crafting_terminal.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -820,8 +786,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:infastone");
-        displayBuilder.setTitle(Text.translate("Infinite Blocks!"));
-        displayBuilder.setDescription(Text.translate("Gaia zum mitnehmen"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.infablocks.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.infablocks.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -844,12 +810,10 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:stargaze_singularity_casing");
-        displayBuilder.setTitle(Text.translate("Power of the Celestials"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.stargaze_singularity_casing.title'});
         displayBuilder.setAnnounceToChat(false);
         displayBuilder.setFrameType("challenge");
-        displayBuilder.setDescription(
-          Text.translate("Hier beginnt der Weg zu ultimativer Macht!")
-        );
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.stargaze_singularity_casing.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -867,8 +831,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:refined_radiance");
-        displayBuilder.setTitle(Text.translate("Refined Situation"));
-        displayBuilder.setDescription(Text.translate("Gespeichertes Licht"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.refined_radiance.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.refined_radiance.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -885,8 +849,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_helmet");
-        displayBuilder.setTitle(Text.translate("Odins Helmet"));
-        displayBuilder.setDescription(Text.translate("Vater der Weisheit!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_helmet.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_helmet.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -904,8 +868,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_chestplate");
-        displayBuilder.setTitle(Text.translate("Zephyrs Chestplate"));
-        displayBuilder.setDescription(Text.translate("Wind der Freiheit!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_chestplate.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_chestplate.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -923,8 +887,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_leggings");
-        displayBuilder.setTitle(Text.translate("Mercurys Leggings"));
-        displayBuilder.setDescription(Text.translate("Listiger Handel!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_leggings.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_leggings.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -942,10 +906,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_boots");
-        displayBuilder.setTitle(Text.translate("Hermes Boots!"));
-        displayBuilder.setDescription(
-          Text.translate("Schneller als dein Internet!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_boots.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_boots.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -963,8 +925,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:netherite_ingot");
-        displayBuilder.setTitle(Text.translate("Compressed Debris"));
-        displayBuilder.setDescription(Text.translate("Frisch aus der Hölle"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.netherite_ingot.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.netherite_ingot.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -981,10 +943,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:stargaze_plate");
-        displayBuilder.setTitle(Text.translate("Material from outer Space"));
-        displayBuilder.setDescription(
-          Text.translate("Macht der Mondnebel, macht auf!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.stargaze_plate.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.stargaze_plate.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1002,8 +962,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:blaze_gold");
-        displayBuilder.setTitle(Text.translate("Blaze in a Box"));
-        displayBuilder.setDescription(Text.translate("Die Seele einer Blaze!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.blaze_gold.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.blaze_gold.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1020,10 +980,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:blaze_plate");
-        displayBuilder.setTitle(Text.translate("Power of the Blazes"));
-        displayBuilder.setDescription(
-          Text.translate("Aus Prometheus Schmiede")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.blaze_plate.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.blaze_plate.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1043,10 +1001,8 @@
       advBuilder
         .display((displayBuilder) => {
           displayBuilder.setIcon("kubejs:blaze_plate_rod");
-          displayBuilder.setTitle(Text.translate("Compact Blazepower"));
-          displayBuilder.setDescription(
-            Text.translate("Der Stab strahlt Wärme aus")
-          );
+          displayBuilder.setTitle({translate: 'cl5.advjs.item.blaze_plate_rod.title'});
+          displayBuilder.setDescription({translate: 'cl5.advjs.item.blaze_plate_rod.description'});
           displayBuilder.setHidden(false);
           displayBuilder.setFrameType("challenge");
         })
@@ -1065,10 +1021,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_sword");
-        displayBuilder.setTitle(Text.translate("Creator Sword"));
-        displayBuilder.setDescription(
-          Text.translate("Damit wird der Enderdrache zum Küken!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_sword.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_sword.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1086,10 +1040,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_shovel");
-        displayBuilder.setTitle(Text.translate("Creator Shovel"));
-        displayBuilder.setDescription(
-          Text.translate("Schaufel dir dein eigenes ... Erdimperium!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_shovel.titl'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_shovel.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1107,10 +1059,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_pickaxe");
-        displayBuilder.setTitle(Text.translate("Creator Pickaxe"));
-        displayBuilder.setDescription(
-          Text.translate("Damit lässt es sich leicht bauen!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_pickaxe.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_pickaxe.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1128,10 +1078,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_axe");
-        displayBuilder.setTitle(Text.translate("Creator Axe"));
-        displayBuilder.setDescription(
-          Text.translate("Baumfäller lieben dieses Tool!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_axe.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_axe.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1149,10 +1097,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:creator_hoe");
-        displayBuilder.setTitle(Text.translate("Creator Hoe"));
-        displayBuilder.setDescription(
-          Text.translate("Dieses Item spricht für Reichtum!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creator_hoe.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creator_hoe.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("challenge");
       })
@@ -1170,8 +1116,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:mithril_ingot");
-        displayBuilder.setTitle(Text.translate("Dwarves mystical Metal"));
-        displayBuilder.setDescription(Text.translate("Made in Mittelerde"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.mithril_ingot.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.mithril_ingot.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1188,10 +1134,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:end_portal_frame");
-        displayBuilder.setTitle(Text.translate("The End, but different"));
-        displayBuilder.setDescription(
-          Text.translate("So kenne ich das aber nicht!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.dimension.the_end.titl'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.dimension.the_end.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("goal");
       })
@@ -1211,10 +1155,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:forest_ravager");
-        displayBuilder.setTitle(Text.translate("Like the Saw, but fancier"));
-        displayBuilder.setDescription(
-          Text.translate("Geschmiedet aus dem Stahl der Zwerge")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.forest_ravager.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.forest_ravager.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1231,8 +1173,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:dragon_egg");
-        displayBuilder.setTitle(Text.translate("I HAVE THE EGG!!!"));
-        displayBuilder.setDescription(Text.translate("Glückwunsch!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.dragon_egg.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.dragon_egg.description'});
         displayBuilder.setHidden(true);
         displayBuilder.setFrameType("challenge");
       })
@@ -1250,10 +1192,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("kubejs:shimmer_nugget");
-        displayBuilder.setTitle(Text.translate("The SHIMMER!"));
-        displayBuilder.setDescription(
-          Text.translate("Ich hab keinen Shimmer davon!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.shimmer_nugget.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.shimmer_nugget.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1270,8 +1210,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:diorite");
-        displayBuilder.setTitle(Text.translate("Ewwww!"));
-        displayBuilder.setDescription(Text.translate("Stein + Taube"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.diorite.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.diorite.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1288,8 +1228,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:crushing_wheel");
-        displayBuilder.setTitle(Text.translate("Garbage Disposal"));
-        displayBuilder.setDescription(Text.translate("Weg mit dem Dreck!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.quartz_dust.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.quartz_dust.description'});
         displayBuilder.setHidden(true);
       })
       .criteria((criteriaBuilder) => {
@@ -1304,10 +1244,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:poppy");
-        displayBuilder.setTitle(Text.translate("Nice try!"));
-        displayBuilder.setDescription(
-          Text.translate("Der Bre hat aber nur Blümchen dabei!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.kill.iron_golem.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.kill.iron_golem.description'});
         displayBuilder.setHidden(true);
       })
       .criteria((criteriaBuilder) => {
@@ -1327,10 +1265,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("amazingtrading:create_shredder");
-        displayBuilder.setTitle(Text.translate("Capitalism Final Stage!"));
-        displayBuilder.setDescription(
-          Text.translate("Ab in den Schredder mit dem Überfluss!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.create_shredder.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.create_shredder.description'});
         displayBuilder.setHidden(false);
         displayBuilder.setFrameType("goal");
       })
@@ -1347,10 +1283,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("createbigcannons:cannon_builder");
-        displayBuilder.setTitle(Text.translate("Arming!"));
-        displayBuilder.setDescription(
-          Text.translate("Groß muss die Kanone sein")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.cannon_builder.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.cannon_builder.description'});
         displayBuilder.setHidden(true);
       })
       .criteria((criteriaBuilder) => {
@@ -1366,8 +1300,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:encased_fan");
-        displayBuilder.setTitle(Text.translate("Better Smelting"));
-        displayBuilder.setDescription(Text.translate("Kohle ist von Gestern"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.encased_fan.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.encased_fan.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1383,10 +1317,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_dd:industrial_fan");
-        displayBuilder.setTitle(Text.translate("Industrial Smelting"));
-        displayBuilder.setDescription(
-          Text.translate("Einer Feuernation würdig!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.industrial_fan.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.industrial_fan.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1402,10 +1334,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:sniffer_egg");
-        displayBuilder.setTitle(Text.translate("He Sniffs everything!"));
-        displayBuilder.setDescription(
-          Text.translate("Da sind Items im Gras verbuddelt?")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.sniffer_egg.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.sniffer_egg.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1421,8 +1351,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:kelp");
-        displayBuilder.setTitle(Text.translate("Jackpot!"));
-        displayBuilder.setDescription(Text.translate("Endlich Fließbänder!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.kelp.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.kelp.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1435,10 +1365,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("minecraft:bamboo");
-        displayBuilder.setTitle(Text.translate("Jungle Fever"));
-        displayBuilder.setDescription(
-          Text.translate("Damit soll ich Zucker machen?")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.bamboo.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.bamboo.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1451,10 +1379,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("vegandelight:wild_soybean");
-        displayBuilder.setTitle(Text.translate("Vegan Challenge"));
-        displayBuilder.setDescription(
-          Text.translate("Auf die Sojabohne, fertig, los!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.soybean.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.soybean.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1471,10 +1397,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("vegandelight:tofu");
-        displayBuilder.setTitle(Text.translate("Tofu à la carte"));
-        displayBuilder.setDescription(
-          Text.translate("Hieraus kann mal alles machen!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.tofu.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.tofu.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1488,10 +1412,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("vegandelight:leather_substitute");
-        displayBuilder.setTitle(Text.translate("Vegan Leather Seats!"));
-        displayBuilder.setDescription(
-          Text.translate("Pflegeleichter als das Original")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.leather_substitute.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.leather_substitute.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1507,10 +1429,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:blaze_burner");
-        displayBuilder.setTitle(Text.translate("MORE HEAT!!"));
-        displayBuilder.setDescription(
-          Text.translate("Heiß, heißer, Blazeburner!")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.blaze_burner.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.blaze_burner.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1526,8 +1446,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create:blaze_cake");
-        displayBuilder.setTitle(Text.translate("It's Over 9000!"));
-        displayBuilder.setDescription(Text.translate("Nur Plasma ist heißer!"));
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.blaze_cake.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.blaze_cake.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1543,10 +1463,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("constructionwand:infinity_wand");
-        displayBuilder.setTitle(Text.translate("Wandalism!"));
-        displayBuilder.setDescription(
-          Text.translate("What do you Wand from me?")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.infinity_wand.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.infinity_wand.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
@@ -1562,10 +1480,8 @@
     advBuilder
       .display((displayBuilder) => {
         displayBuilder.setIcon("create_sa:creative_filling_tank");
-        displayBuilder.setTitle(Text.translate("Unlimited Jetpack Fuel!"));
-        displayBuilder.setDescription(
-          Text.translate("Von wegen H² ist die Zukunft")
-        );
+        displayBuilder.setTitle({translate: 'cl5.advjs.item.creative_filling_tank.title'});
+        displayBuilder.setDescription({translate: 'cl5.advjs.item.creative_filling_tank.description'});
         displayBuilder.setHidden(false);
       })
       .criteria((criteriaBuilder) => {
